src/app/services/blockchain.service.ts

```diff
@@ -62,15 +62,15 @@ export class BlockchainService {
   }
 
   private loadBlockchainBlocks() {
-    setTimeout(() => IntervalObservable
-      .create(2000)
+    IntervalObservable
+      .create(90000)
       .flatMap(() => this.getBlockchainProgress())
```

Comment:
This needs an additional change. As IntervalObservable does not run for the first time immediately, but after the established time has elapsed, it is now necessary to wait 3 minutes until the wallet shows the balance. This should not happen, the connection must be made immediately. One option is to put `.startWith(1)` just below `.create(90000)`.